storybook-example/version_build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
No, we can not have a Linux script to determine the version...

Can we not add it as a npm custom command?

Reply:
If you have git bash in windows, then these command will work there perfectly. I have run these command in my windows machine with the help of git bash only. 

Another thing, this command will use only for deployment purpose and not for locally running the storybook

Reply:
let me look into this @gsasikumar

Reply:
hi @gsasikumar we have remove shell script and added the javascript in this [d07328a](https://github.com/mosip/mosip-plugins/pull/58/commits/d07328a1fc4c5e9df7714782776fdf3db330d6f7)